src/services/api/bookCopyService.ts

```diff
@@ -0,0 +1,18 @@
+import axios from '../../axios/axiosConfig';
+
+interface BookCopy {
+  id: null;
+  identifier: string;
+  bookId: string | undefined;
+}
+
+const createBookCopy = (id: string | undefined, data: BookCopy) => {
+  return axios
+    .post(`/book/${id}/book-copy`, data)
```

Comment:
What are you doing here if id is undefined, for me it is better not to use undefined